xconf-angular-admin/src/main/webapp/app/xconf/config/state.config.js

```diff
@@ -391,13 +391,13 @@
                     permissions: [PERMISSION.WRITE_TELEMETRY_ALL, PERMISSION.WRITE_TELEMETRY_STB, PERMISSION.WRITE_TELEMETRY_XHOME, PERMISSION.WRITE_TELEMETRY_RDKCLOUD]
                 }
             })
-            .state('telemetrytwotargetingrule-view', {
-                controller: 'TelemetryTwoTargetingRuleViewController',
+            .state('telemetrytwotargetingrule-import', {
+                controller: 'TelemetryTwoTargetingRuleImportController',
                 controllerAs: 'vm',
-                url: '/telemetrytwo/targetingrule/view/:ruleId',
-                templateUrl: 'app/xconf/telemetry/telemetrytwotargetingrule/telemetrytwotargetingrule.view.html',
+                url: '/telemetrytwo/targetingrule/import',
+                templateUrl: 'app/xconf/telemetry/telemetrytwotargetingrule/telemetrytwotargetingrule.import.html',
                 data: {
-                    permissions: [PERMISSION.READ_TELEMETRY_ALL, PERMISSION.READ_TELEMETRY_STB, PERMISSION.READ_TELEMETRY_XHOME, PERMISSION.READ_TELEMETRY_RDKCLOUD]
+                    permissions: [PERMISSION.WRITE_TELEMETRY_ALL, PERMISSION.WRITE_TELEMETRY_STB, PERMISSION.WRITE_TELEMETRY_XHOME, PERMISSION.WRITE_TELEMETRY_RDKCLOUD]

```

Comment:
Same here. Use PERMISSION.WRITE_TELEMETRY_PERMISSIONS. It's also relevant for all pages related to Telemtry 2.0